openeo/extra/job_management.py

```diff
@@ -192,24 +192,17 @@ def _normalize_df(self, df: pd.DataFrame) -> pd.DataFrame:
             ("duration", None),
             ("backend_name", None),
         ]
-        new_columns = {
-            col: val for (col, val) in required_with_default if col not in df.columns
-        }
+        new_columns = {col: val for (col, val) in required_with_default if col not in df.columns}
         df = df.assign(**new_columns)
-        # Workaround for loading of geopandas "geometry" column.
-        if "geometry" in df.columns and df["geometry"].dtype.name != "geometry":
-            df["geometry"] = df["geometry"].apply(shapely.wkt.loads)
-        return df
 
-    def _persists(self, df, output_file):
-        df.to_csv(output_file, index=False)
-        _log.info(f"Wrote job metadata to {output_file.absolute()}")
+        return df
 
     def run_jobs(
         self,
         df: pd.DataFrame,
         start_job: Callable[[], BatchJob],
         output_file: Union[str, Path],
+        read_write_helper=None,
```

Comment:
In this first PR I would not provide this `read_write_helper` yet.
Let's first make sure that we get parquet support right, before committing to a new user facing API